modules/microshift-manifests-overview.adoc

```diff
@@ -4,12 +4,23 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="microshift-manifests-overview_{context}"]
-= How manifests work with kustomize
+= How kustomize works with manifests to deploy applications
 
-The `kustomize` configuration management tool is integrated with {microshift-short}. At every start, {microshift-short} searches the `/etc/microshift/manifests`, `/etc/microshift/manifests.d/++*++`, `/usr/lib/microshift/`, and `/usr/lib/microshift/manifests.d/++*++` manifest directories for a `kustomization.yaml`, `kustomization.yml`, or `Kustomization` file. If it finds one, {microshift-short} automatically runs the equivalent of the `kubectl apply -k` command to apply the identified manifests to the cluster.
+The `kustomize` configuration management tool is integrated with {microshift-short}. You can use Kustomize and the {oc-first} together to apply customizations to your application manifests and deploy those applications to a {microshift-short} cluster.
+
+* A `kustomization.yaml` file is a specification of resources plus customizations.
+* Kustomize uses a `kustomization.yaml` file to load a resource, such as application, then applies any changes you want to that application manifest and produces a copy of the manifest with the changes overlaid.
+* This operation keeps the original configuration file for your application intact, while enabling you to deploy iterations and customizations of your applications efficiently.
```

Comment:
This seems to be a continuation of the second bullet, rather than a new one. Consider one of these approaches:
```suggestion
* Kustomize uses a `kustomization.yaml` file to load a resource, such as application, then applies any changes you want to that application manifest and produces a copy of the manifest with the changes overlaid. This operation keeps the original configuration file for your application intact, while enabling you to deploy iterations and customizations of your applications efficiently.
```
or
```suggestion
* Kustomize uses a `kustomization.yaml` file to load a resource, such as application, then applies any changes you want to that application manifest and produces a copy of the manifest with the changes overlaid. 
+
This operation keeps the original configuration file for your application intact, while enabling you to deploy iterations and customizations of your applications efficiently.
```